CONTRIBUTING.adoc

```diff
@@ -70,8 +70,13 @@ Please carefully follow the whitespace and formatting conventions already presen
 Whitespace management tips
 
 . You can use the https://marketplace.eclipse.org/content/anyedit-tools[AnyEdit Eclipse plugin] to ensure spaces are used and to clean up trailing whitespaces.
+  Once installed this plugin an extra mandatory configuration must be completed to accomplish the _Unix (LF), not dos (CRLF) line endings_ mentioned in the previous list go to: `Preferences` -> `General` -> `Editors` -> `AnyEdit Tools` -> `Auto Convert` tab and check the `Fix line delimiters (CRLF to LF etc)` checkbox.
```

Comment:
I think this paragraph is a candidate for backporting to additional versions. As such will you please separate this paragraph into a separate commit from the Spring JavaFormat changes?

Reply:
>As such will you please separate this paragraph into a separate commit from the Spring JavaFormat changes?

Not sure if I am following you. Can you expand the idea pls?